success-criteria.md

```diff
@@ -337,21 +309,23 @@ stakeholder group.
 
         -   Free elections protected against foreign manipulation
 
-            -   While free speech requires allowing speech the majority does not
-                approve of, we can label political speech by the author’s
-                nationality and whether it is endorsed by one or more candidates
-
-        -   Freedom of the press enables watch-dog reporting on important issues
-            and combats fake news
-
     -   Freedom of information to provide fast, easy access to internet-enabled
         content for all
 
         -   Cost-free access enables access for all, regardless of economic
             means
 
-        -   Freedom from having to self-censor for fear of content consumption
-            being associated with directly-identifiable, offline identity
+-   Freedom of the press, which enables and supports:
+ 
+	- watch dog reporting to keep governments and companies transparent and accountable
+  
+	- combating fake news
+  
+	- refuting and providing resources to counteract hoaxes, fraud, and libel
+  
+	- support and amplification for citizen interests
+
+    - Freedom of press includes freedom for publications to amplify speech the majority does not approve of, however freedom of press and speech is curtailed unless all content is clearly accredited to specific publishers and authors and publications have the ability to distinguish themselves and their authors by making metadata about themselves and their authors available and transparent to the reader.
```

Comment:
The above changes coincide well with efforts by the Schema dot org group to add additional dimensions of data about policy and objectives to both authors and publishers.